srcpkgs/wvkbd/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'wvkbd'
+pkgname=wvkbd
+version=0.7
+revision=1
+build_style=gnu-makefile
+hostmakedepends="pango-devel cairo-devel wayland-devel pkg-config"
+makedepends="fontconfig-devel libxkbcommon-devel"
+short_desc="On-screen keyboard for wlroots"
+maintainer="Eloi Torrents <eloi.torrentsj@gmail.com>"
+license=GPL-3.0-only
+homepage=https://github.com/jjsullivan5196/wvkbd
+distfiles=https://github.com/jjsullivan5196/wvkbd/archive/refs/tags/v${version}.tar.gz
+checksum=b5d3c8bd97ef6cdd962fcee2ca9593f16569dca322a89c408d704718735cc18e
+
+do_build() {
+ make CC=$CC INCS="-I. -I${XBPS_CROSS_BASE}/usr/include/pango-1.0/pango"
```

Comment:
the `INCS` is not necessary, and the `CC` can be put in `make_build_args` instead of override the build phase

Reply:
I tried `make_build_args="CC=$CC"` but I get this error:
```
drw.h:4:10: fatal error: pango/pangocairo.h: No such file or directory
    4 | #include <pango/pangocairo.h>
      |          ^~~~~~~~~~~~~~~~~~~~
```

Reply:
try running `./xbps-src clean` then try again. I got that error sometimes when building this with an unclean masterdir state